README.md

```diff
@@ -19,13 +19,14 @@ If the command you are trying to use is not compatible, `pip_api` will raise a
 * `pip_api.version()`
   > Returns the `pip` version as a string, e.g. `"9.0.1"`
 
-* `pip_api.installed_distributions(local=False)`
+* `pip_api.installed_distributions(local=False, paths=[])`
```

Comment:
Since technically this isn't "available with all `pip` versions" anymore, I'm trying to decide if we should move this to a new section "Available with `pip>=19.2`", just add a note here, or something else. Thoughts?

Reply:
How's this? I list it twice but only document the `paths` parameter under the `pip>=19.2` heading.

d25b144